README.md

```diff
@@ -4,13 +4,14 @@ A bridge between [Lichess API](https://lichess.org/api#tag/Chess-Bot) and bots.
 
 ## How to Install
 - NOTE: Currently, only Python3 is supported
+- (For convinience of Windows users) Here is a video on how to install the bot: (https://youtu.be/AuncFctntYs). Or you can proceed to these steps. If you don't have Python, you may download it here: (https://www.python.org/downloads/). When installing it, enable "add Python to PATH", then go to custom installation and enable all options (especially "install for all users"), except the last. It's better to install Python in a path without spaces, like "C:\Python\. To type commands it's better to use PowerShell. Go to Start menu and type "PowerShell". Then you may need to upgrade pip. Execute "python -m pip install --upgrade pip" in PowerShell. 
```

Comment:
The formatting of this needs to improve. Make 2 sections. One for WIndows installation and one for Mac/Linux